server/api/branch/tests.py

```diff
@@ -33,3 +37,53 @@ def test_GetAllExceptis_deleted(self):
         self.branch.delete()
         self.branch2.delete()
         self.assertEqual(len(Branch.objects_deleted.all()), 2)
+
+
+ADMIN = "adminTest2"
+ADMIN_PASS = "adminTestPassword2"
+
+
+USER = "normalUser2"
+USER_PASS = "normalUserPassword2"
+
+
+class BranchAuthTest(TestCase):
+
+    def setUp(self) -> None:
+        self.client = APIClient()
+        self.adminUser: User = User.objects.create_user(
+            username=ADMIN, password=ADMIN_PASS, is_staff=True
+        )
+        self.url = reverse("branch-list")
+        self.user: User = User.objects.create_user(username=USER, password=USER_PASS)
+        self.postData = {
+            "name": "My Branch",
+            "description": "This is a description of my Branch.",
+        }
+
```

Comment:
Should add a test for authentication as well, because most of the endpoints require authentication